algorithms/symmetry/cosym/__init__.py

```diff
@@ -370,28 +372,6 @@ def _analyse_symmetry(self):
         )
         self.params.cluster.n_clusters = len(cosets.partitions)
 
-    def _space_group_for_dataset(self, dataset_id, sym_ops):
```

Comment:
You're not wrong that retaining this adds confusion, as I was trying to work out how things still work with all this stuff removed...